hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -329,4 +329,8 @@ private OzoneConsts() {
   public static final String GDPR_ALGORITHM = "algorithm";
 
 
+  // Transaction Info
+  public static final String TRANSACTION_INFO_KEY = "TRANSACTIONINFO";
+  public static final String TRANSACTION_INFO_SPLIT_KEY = "-";
+
```

Comment:
Question out of curiosity: In another patch (HDDS-3217), the Keys in DB are suffixed with a #. Is that a just convention followed for ContainerDBs or has any significance?

Reply:
Technically we don't need it. As in container DB we have iterators to iterate the keys in container, but there are some keys like BCSID, blockCount which are not blocks, so to avoid this they have a filter to skip these keys during iteration (But the same can be done without # also. Might be there it is done to know that these are system defined keys and a way to distinguish. (This is just my understanding, and followed the same when implementing HDDS-3217). In OM, we have tables for each. But I can use the # prefix.

Reply:
Done.